### DISCUSSION TITLE
License agreements

### DISCUSSION THREAD

**Discussion by Lienes** [Tencent/ncnn#4749] [General]
Hi, 

Can anyone help with explanation with license agreements. 
Can I use it for commercial purpose ? What about deployment API ? Do there is some special cases or requirements ? 

Thanks in advance :)

**Comment by nihui** [comment]
ncnn is an open source project released under the bsd-3 license, like opencv etc
You can use it for commercial purpose
For details, refer to the bsd-3 license
